src/criterion/BranchObjectiveFunction.ts

```diff
@@ -99,50 +100,33 @@ export class BranchObjectiveFunction<T extends Encoding>
     const branchNode = this._subject.cfg.nodes.find((n: Node) => {
       return n.type === NodeType.Branch && n.lines.includes(this._line);
     });
+
+    // TODO maybe childNode is not required.
```

Comment:
We could calculate the approach level from the branch node. This conceptually makes more sense.